src/renderer/families/tezos/Delegation/ContextMenu.js

```diff
@@ -63,6 +64,7 @@ const ContextMenu = ({ account, parentAccount }: Props) => {
           openModal("MODAL_DELEGATE", {
             parentAccount,
             account,
+            mode: "redelegate",
```

Comment:
⚠️  the issue is here. we must use another concept. note naming this "mode", because "mode" is already used somewhere to actually set the transaction **mode**.. because what I can see from @Arnaud97234 's log is that we have this in transaction mode

<img width="228" alt="Capture d’écran 2020-11-10 à 17 18 35" src="https://user-images.githubusercontent.com/211411/98700762-d7e3a480-2378-11eb-800a-f8b2ca89448c.png">

which is not a valid transaction mode at all

https://github.com/LedgerHQ/ledger-live-common/blob/master/src/families/tezos/types.js#L30

which makes the transaction trying a "Send" on an empty recipient. therefore the error

so the bug is along this path.